Foundation/Scanner.swift

```diff
@@ -579,6 +589,7 @@ extension Scanner {
         }
     }
     
+    @discardableString
```

Comment:
Looks like a typo, should be `@discardableResult`.

Reply:
Oh. Really stupid typo. Fixed